save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -243,8 +245,8 @@ class FixPlugin(
         @Serializable(with = PathSerializer::class) val expected: Path
     ) : TestFiles {
         override fun withRelativePaths(root: Path) = copy(
-            test = test.relativeTo(root),
-            expected = expected.relativeTo(root),
+            test = test.createRelativePathToTheRoot(root).toPath(),
```

Comment:
In https://github.com/square/okio/commit/a235589cac329973c4799ce9c6474e0b2ddb81e4#diff-eaa42bf2d76129a0f76061b7b82e8a904546f522cf2947b23207d0c010b7f1d6R260

they disable normalization in `commonRelativeTo` method (and this can't be configured), so we've got an extra `../` in our reporters
the calling of `normalized()` after `relativeTo` didn't change anything, I also didn't find any other canonization methods, so I restore our realization

Reply:
In [release notes](https://github.com/diktat-static-analysis/save-cloud/pull/501/commits/b71b58d614d2733e0be77629fe2eafc160f1e127) they state that `Path.normalize()` or `FileSystem.canonicalize()` can be used, have you tried these?